services/API-service/src/api/rainfall-triggers/rainfall-triggers.entity.ts

```diff
@@ -0,0 +1,43 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+@Entity('rainfallTriggers')
+export class RainfallTriggersEntity {
+  @PrimaryGeneratedColumn('uuid')
+  public id: string;
+
+  @Column()
```

Comment:
`countryCode` here accepts a random string

It should only accept values from the country table. [See example here](https://github.com/rodekruis/IBF-system/blob/7f70ea0389a97db0e5efa34afe31b2fc6d8ab3c2/services/API-service/src/api/admin-area-data/admin-area-data.entity.ts#L15-L17).

Reply:
@gulfaraz but then how do I code this in the seed-script? I first need to get the full countryEntity, and pass that to the country-column in the rainfallTriggersEntity?

Reply:
It should be possible to do this without fetching the `countryEntity`.
The definition in the `rainfallTriggersEnitity` is updated but the rest should work as is.

We're only using a _referenced string_ instead of _any string_ for `countryCode`.